src/components/node/stanford-person/card.tsx

```diff
@@ -1,23 +1,111 @@
-"use client"
-
-import VerticalPersonCard from "./vertical-card"
-import HorizontalPersonCard from "./horizontal-card"
+import Image from "next/image"
+import Link from "@/components/patterns/elements/drupal-link"
+import {EnvelopeIcon} from "@heroicons/react/20/solid"
+import LibCal from "./libcal"
+import EmailLink from "@/components/patterns/elements/email-link"
+import {buildUrl} from "@/lib/drupal/utils"
 import {NodeStanfordPerson} from "@/lib/gql/__generated__/drupal.d"
-import {useDebounceCallback, useResizeObserver} from "usehooks-ts"
-import {RefObject, useRef, useState} from "react"
 
-const StanfordPersonCard = ({node, ...props}: {node: NodeStanfordPerson}) => {
-  const ref = useRef<HTMLDivElement>(null)
-  const [{width}, setSize] = useState<{width?: number; height?: number}>({width: undefined})
+interface Props {
+  node: NodeStanfordPerson
+  h3Heading?: boolean
+}
 
-  const onResize = useDebounceCallback(setSize, 200)
-  useResizeObserver({ref: ref as RefObject<HTMLDivElement>, onResize})
+const StanfordPersonCard = ({node, h3Heading, ...props}: Props) => {
+  const HeadingElement = h3Heading ? "h3" : "h2"
+  const imageUrl = node.suPersonPhoto?.mediaImage.url
 
-  const Card = width && width < 510 ? VerticalPersonCard : HorizontalPersonCard
   return (
-    <div ref={ref}>
-      <Card node={node} {...props} />
-    </div>
+    <article className="@container" {...props}>
+      {/* Vertical layout (default) */}
```

Comment:
FYI @pookmish I’m going to check in with Darcy and Sarah to confirm whether the Horizontal Card is still relevant. If it is, I’ll refine and consolidate the styles further. 

Otherwise, I’d love your feedback on the GraphQL when you have time 💰